.github/workflows/tox.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           snap watch --last=install
           echo "::endgroup::"
       - name: Run Linters
-        run: tox run --skip-pkg-install -m lint
+        run: tox run --no-list-dependencies --skip-pkg-install --colored=yes -m lint
```

Comment:
`--no-list-dependencies` isn't automatically being used by default in github actions